### DISCUSSION TITLE
Can you pls let me know if there is file upload component in the Commercetools CDK

### DISCUSSION THREAD

**Discussion by vijaya-sarathy** [commercetools/ui-kit#2705] [Q&A]
Can you pls let me know if there is file upload component in the Commercetools CDK
Something similar to what we see in operation log import upload screen **operations/import/upload**

Thanks

![image](https://github.com/commercetools/ui-kit/assets/71578090/6039b447-046b-484a-a7cf-d13c301b89db)

**Comment by mmaltsev-ct** [comment]
Hi @vijaya-sarathy, unfortunately, we don't provide this component in UI-Kit, but thanks for creating a Discussion, as we might add it in the future then, since it's requested. Could you tell a bit more about your use case, please?
